ext_test.go

```diff
@@ -46,8 +47,10 @@ func TestHungRequest(t *testing.T) {
 		})
 	}
 
-	require.NoError(t, hosts[0].Peerstore().AddProtocols(hosts[1].ID(), protocol.ConvertToStrings(d.serverProtocols)...))
-	d.peerFound(ctx, hosts[1].ID(), true)
+	// Wait at most 100ms for a peer in our routing table.
```

Comment:
This is one second, not 100ms. Would also probably not reference he actual number in the comment because it's likely to go stale